pkg/postgres/postgres.go

```diff
@@ -59,8 +70,12 @@ func LogTrace(ctx context.Context, logger *logging.Logger, contextString string)
 		return
 	}
 	for _, e := range tracer.events {
+		rowsAccessed := 1
```

Comment:
why not `0`?

Reply:
Because if the rowsAccessed are unset then it is a QueryRow call and that will always have one return